scripts/jenkins/cloud/ansible/roles/ardana_deploy/tasks/setup_openid_connect.yml

```diff
@@ -0,0 +1,34 @@
+#
+# (c) Copyright 2019 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+---
+
+- name: Copy OpenID Connect config file
+  copy:
+    src: openid_connect_bogus_idp_config.yml
+    dest: /tmp/openid_connect_bogus_idp_config.yml
+  register: openid_connect_bogus_idp_config
+
+- name: Enable OpenID Connect in Keystone
+  shell: >
+    cat /tmp/openid_connect_bogus_idp_config.yml >> {{ ardana_openstack_path }}/roles/keystone-common/vars/keystone_deploy_config.yml
```

Comment:
This operation is not idempotent, so if this play/action is run multiple times, we get multiple copies of the keystone_openid_connect_conf definition in the target yaml file.

Maybe not a big deal, but can easily be mitigated by either using lineinfile edit to add the necessary lines, or else use some shell wrapping (if ! grep ...) to only append the change if the keystone_openid_connect_conf entry is not already in the file.